Makefile

```diff
@@ -13,3 +13,18 @@ hdf5comp:
 
 %.patch:
 	curl -f -L -O https://src.fedoraproject.org/rpms/hdf5/raw/master/f/$@
+
+test:
+	$(call install_repos,$(NAME)@$(BRANCH_NAME):$(BUILD_NUMBER))
+	yum -y install \
+	  $(NAME) \
+	  java-$(NAME) \
+	  $(NAME)-devel \
+	  $(NAME)-static \
+	  $(NAME)-mpich \
+	  $(NAME)-mpich-devel \
+	  $(NAME)-mpich-static \
+	  $(NAME)-openmpi \
+	  $(NAME)-openmpi-devel \
+	  $(NAME)-openmpi-static \
```

Comment:
We don't build the _openmpi_ variant
https://github.com/daos-stack/hdf5/blob/53d23772fdd680b6893982b10391fd4a33e6fc6d/hdf5.spec#L68

Feel free to change that if you like though.  It was only disabled at the time for expediency of getting the RPMs that we needed specifically out.

Reply:
If we need this enabled I think we should address it in a separate ticket where we can also update the test to include the RPM install verification.